pkg/agent/clientset.go

```diff
@@ -43,6 +44,9 @@ type ClientSet struct {
 	// proxy server.
 	probeInterval time.Duration // The interval by which the agent
 	// periodically checks if its connections to the proxy server is ready.
+	syncIntervalCap time.Duration // The maximum interval
```

Comment:
Would be nice to add validation for probeInterval and syncIntervalCap. (Eg probeInterval < syncIntervalCap)

Reply:
+1

Reply:
Fixed in `cmd/agent/main.go`